internal/worker/worker.go

```diff
@@ -0,0 +1,94 @@
+package worker
+
+import (
+	"context"
+	"sync"
+	"sync/atomic"
+
+	"github.com/etf1/kafka-mongo-watcher/internal/kafka"
+	"github.com/etf1/kafka-mongo-watcher/internal/mongo"
+	"github.com/gol4ng/logger"
+	kafkaconfluent "gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
+)
+
+type Worker interface {
+	Close()
+	Replay(ctx context.Context, collection mongo.CollectionAdapter, topic string)
```

Comment:
same remark than Client
2 method that do "similar/same" job

Reply:
Worker now have only 1 method: `Work()`, no more Replay and Watch